pythonforandroid/recipes/python3/__init__.py

```diff
@@ -23,9 +24,10 @@ class Python3Recipe(GuestPythonRecipe):
     name = 'python3'
 
     patches = ["patches/fix-ctypes-util-find-library.patch"]
-
     if sh.which('lld') is not None:
-        patches = patches + ["patches/remove-fix-cortex-a8.patch"]
+        patches += ["patches/remove-fix-cortex-a8.patch"]
```

Comment:
shortened a little this line...to be more readable again and match the ones I added below